po/uk.po

```diff
@@ -5381,6 +5381,50 @@ msgstr "Пропускаємо %s через попередню помилку"
 msgid "Aborted due to failure (%s)"
 msgstr "Перервано через критичну помилку (%s)"
 
+#: common/flatpak-uri.c:118
+#, no-c-format
+#| msgid "Invalid extension name %s"
+msgid "Invalid %-encoding in URI"
+msgstr "Некоректне %-eкодування в адресі"
+
+#: common/flatpak-uri.c:135
+msgid "Illegal character in URI"
+msgstr "Некоректний символ у а"
```

Comment:
```suggestion
msgstr "Некоректний символ у адресі"
```

(I don't speak Ukrainian but this would match the surrounding translations which also have "URI" in them)

Reply:
Fixed. Many thanks. Sorry for this typo.